### PR TITLE
#160577479 User should be able to create and view their profile

### PR DESCRIPTION
**Pull request by IncredibleBlackMan**
#### What does this PR do?
Have the user profile created on user registration.
Allows a user to update their user profile details.
#### Description of Task to be completed?
Have the following endpoints working:
```GET /api/profiles/<username>/```
```PUT /api/profiles/<username>/```
#### How should this be manually tested?
- After cloning the repo, make migrations and test the above endpoints using postman.
#### Any background context you want to provide?
- The username to be tested needs to be registered first for this to work
#### What are the relevant pivotal tracker stories?
#160577479
#### Any Necessary Screenshots
screen shot 2018-10-10 at 15 06 03

### CODE REVIEW

File: authors/urls.py [hunk 1]
```diff
@@ -20,5 +20,6 @@
     path('admin/', admin.site.urls),
 
     path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication')),
+    path('api/', include(('authors.apps.profiles.urls', 'authentication'), namespace='profiles')),
```
Comment (codingedward) [inline authors/urls.py]: Check the second parameter passed in the `include`  tuple. From [Django's documentation](https://docs.djangoproject.com/en/2.1/ref/urls/#include), it ought to be:
```python
 include((pattern_list, app_namespace), namespace=None)
```
Just change the `authentication` to `profiles` and we'll be okay on this.

**Review by codingedward — CHANGES_REQUESTED** [review]
Hi Ian, 
As we agreed to have TDD, could we kindly have tests for the profile feature? You could simply base it on the tests on the `authentication` app.

Reach out in case of any help needed.

File: authors/settings.py [hunk 2]
```diff
@@ -152,6 +152,3 @@
 
 # Activate django-heroku for Heroku
 django_heroku.settings(locals())
-
```
Comment (koechkevin) [inline authors/settings.py]: I don't understand why you removed this

**Review by koechkevin — COMMENTED** [review]
looks good

File: authors/apps/authentication/backends.py [hunk 3]
```diff
@@ -1,13 +1,44 @@
...
-# from rest_framework import authentication, exceptions
-#
-# from .models import User
+from django.conf import settings
+from rest_framework import authentication
+from rest_framework.exceptions import AuthenticationFailed
+from .models import User
+
 
 """Configure JWT Here"""
```
Comment (lukorito) [inline authors/apps/authentication/backends.py]: We should probably remove this

**Review by codingedward — APPROVED** [review]

**Comment by codingedward** [comment]
@FawazFarid, I believe this is ready for a merge.

**Comment by IncredibleBlackMan** [comment]
@FawazFarid I believe this is ready for merge after you have reviewed it.

**Comment by koechkevin** [comment]
This PR is ready for a merge.

**Pull request merged**